### DISCUSSION TITLE
How to extract vector graph

### DISCUSSION THREAD

**Discussion by LastMonody** [pymupdf/PyMuPDF#1447] [Q&A] [answered]
I have a scanned PDF, but I pass the page.getImageList(full = true) cannot determine whether the PDF can be parsed

**Comment by JorjMcKie** [comment]
This is of course **_no bug!_**
Obviously meant as a question - please use the `Discussions` tab next time.

Your formulation is however unclear: does "parsing" mean whether you can extract text?
A scanned page always is a **_raster_** image, so asking for a **_vector_** image makes no sense.

**Comment by LastMonody** [comment]
Sorry, the first time I submitted a question, the wrong question became a bug. First, my purpose is to pass the page Getimagelist (full = true) determines whether there are pictures on the page. If there are pictures, I will parse them through OCR. If not, I will parse them through Fitz. But now the hand I call this method is empty, so I use Fitz to extract text, but it is empty. I also try two modes: vector image and bitmap, but bitmap can extract images, However, the page is a vector image, but the image cannot be obtained

**Comment by JorjMcKie** [comment, marked as answer]
I answered in the new discussion ...
